main.js

```diff
@@ -147,3 +148,9 @@ function checkErrorMessages() {
     }
   }
 }
+
+function replaceTimer() {
+timer.textContent = `YOU DID IT! CONGRATULATIONS ON FINISHING YOUR ${createdActivities[0].category.toUpperCase()} SESSION!`;
+timer.classList.add('timer-removed');
+display(logActivityButton);
+}
```

Comment:
Maybe we can give this a different name like "displayMessage" for readability it seems more descriptive of what is going on in this function.

Reply:
For sure!